crates/sui-sdk/src/wallet_context.rs

```diff
@@ -337,235 +332,4 @@ impl WalletContext {
             )
             .await?)
     }
-
```

Comment:
@emmazzz -- can you check my working here? I think you made the change to move these functions here.  

I can't remember what the order of operations were (whether the helper functions came first and then we moved `WalletContext` into the SDK or vice versa), but it seems strange to include these in our publicly facing SDK, and they introduce a way to bypass execution versioning that was caught by the test I added.

These helpers only using publicly available parts of `WalletContext`, so I've moved them into `sui-test-transaction-builder`.